ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/rules/MultiLineIfElseRule.kt

```diff
@@ -64,7 +64,10 @@ public class MultiLineIfElseRule :
             return
         }
 
-        if (node.elementType == ELSE && node.firstChildNode?.elementType == BINARY_EXPRESSION) {
+        if (node.elementType == ELSE &&
+            node.firstChildNode?.elementType == BINARY_EXPRESSION &&
+            node.firstChildNode.firstChildNode?.elementType == IF
```

Comment:
Line 69 is confusing given the code sample on line 71-76.

Reply:
Updated